### PR TITLE
Fgr 3641 add shipping enmu

### PR DESCRIPTION
**Pull request by Mikewa33**

### CODE REVIEW

**Comment by hf2186** [comment]
I feel like this is missing something. Where is the limiting selection logic when doing the mapping?

File: db/migrate/20200828174620_add_shipping_option_filter_to_mirakl_store.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddShippingOptionFilterToMiraklStore < ActiveRecord::Migration
+  def change
+    add_column :spree_mirakl_stores, :shipping_method_display_rate, :string, default: 'both'
```
Comment (hf2186) [inline db/migrate/20200828174620_add_shipping_option_filter_to_mirakl_store.rb:3]: Shouldn't this be an int? Why is this text?

**Review by hf2186 — CHANGES_REQUESTED** [review]

File: db/migrate/20200828174620_add_shipping_option_filter_to_mirakl_store.rb (hunk 1 above)
Comment (Mikewa33) [reply db/migrate/20200828174620_add_shipping_option_filter_to_mirakl_store.rb:3]: This is how spree does it. For some reason, their dropdown maps to 
```DISPLAY = [:both, :front_end, :back_end]

    # Used for #refresh_rates
    DISPLAY_ON_FRONT_AND_BACK_END = 0
    DISPLAY_ON_FRONT_END = 1
    DISPLAY_ON_BACK_END = 2```
so we use display like they do and then map it to the const values

File: app/models/spree/mirakl_store.rb [hunk 2]
```diff
@@ -34,5 +34,18 @@ def get_carriers_from_mirakl
         Spree::MiraklStoreCarrier.find_or_create_by(label: carrier[:label].downcase, code: carrier[:code], mirakl_store: self)
       end
     end
+
+    def return_shipping_method_filter
+      case self.shipping_method_display_rate
+      when 'both'
```
Comment (hf2186) [inline app/models/spree/mirakl_store.rb:40]: remove both and use else block

**Review by hf2186 — CHANGES_REQUESTED** [review]
minor change

**Review by hf2186 — APPROVED** [review]

**Pull request merged**